test-resources/headless-core-stub/utils/src/services/client-configuration.ts

```diff
@@ -1,14 +1,16 @@
+import { STUB_PRIVATE_SIGNING_KEY_PARAMETER_PATH } from "../constants";
 import { getParametersValues } from "../parameter/get-parameters";
 import config from "./config";
-const { commonParameterPrefix, testResourcesParameterPrefix } = config;
+const { commonStackName } = config;
 
 export class ClientConfiguration {
     public static async getConfig(clientId: string) {
         const parameters = [
-            `/${commonParameterPrefix}/clients/${clientId}/jwtAuthentication/audience`,
-            `/${commonParameterPrefix}/clients/${clientId}/jwtAuthentication/issuer`,
-            `/${commonParameterPrefix}/clients/${clientId}/jwtAuthentication/redirectUri`,
-            `/${testResourcesParameterPrefix}/ipv-core-stub-aws-headless/privateSigningKey`,
+            `/${commonStackName}/clients/${clientId}/jwtAuthentication/audience`,
+            `/${commonStackName}/clients/${clientId}/jwtAuthentication/issuer`,
+            `/${commonStackName}/clients/${clientId}/jwtAuthentication/redirectUri`,
+            `/${commonStackName}/clients/${clientId}/jwtAuthentication/publicSigningJwkBase64`,
+            STUB_PRIVATE_SIGNING_KEY_PARAMETER_PATH,
```

Comment:
Why are we not following the same pattern for this as the other SSM params?

Reply:
It an absolute path parameter just like https://github.com/govuk-one-login/ipv-cri-pipeline-deployment/blob/main/config/check-hmrc/build/ssm/main.tf#L22-L40

Reply:
Sorry maybe I wasn't clear - what is the reason are we not following the pattern this is being used in this file for other SSM params?

Reply:
You mean  this pattern`/${testResourcesParameterPrefix}/${clientId}/privateSigningKey` 
As mention the parameter is really an absolute one all stubs stacks would be using it. Include one the you would create you had you test-resources stack for instance, It does change stack by stack

Reply:
The parameter could have been named `/ipv-core-stub-aws-headless/privateSigningKey` it is currently used in `dev, build` by ours stubs and cross all other flavoured existing stub environment stacks

Reply:
Right ok you're saying that the SSM param isn't deployed with the test-harness stack itself so using the name of the test-harness stack isn't helpful? 

FIne, happy for us to hard code "test-resources" then but let's do it in this file rather than as a constant since it's not reused and we need the `clientId` override since that is possible to override everywhere else in the stub.